rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ConfigManagementServiceImpl.java

```diff
@@ -196,6 +196,17 @@ public void removeConnectorConfig(String connectorName) {
         sendSynchronizeConfig();
     }
 
+    @Override
+    public void deleteConnectorConfig(String connectorName) {
```

Comment:
It would be better to add the verification of whether this connector is stopped.